pkgs/development/python-modules/tifffile/default.nix

```diff
@@ -23,11 +24,12 @@ buildPythonPackage rec {
   '';
 
   propagatedBuildInputs = [ numpy ]
-    ++ lib.optional isPy27 [ futures enum34 pathlib ];
+    ++ lib.optional isPy27 [ futures enum34 pathlib ]
+    ++ lib.optional isPy3k [ imagecodecs-lite ];
```

Comment:
Why is `imagecodecs-lite` only needed for py3k?

Reply:
@JohnAZoidberg due to imagecodecs-lite being broken on python2: https://github.com/NixOS/nixpkgs/blob/6306baa8a2ad1f0db0994bc2a05278d0d3c8ffca/pkgs/top-level/python-packages.nix#L2627 and https://github.com/NixOS/nixpkgs/pull/62754#issuecomment-528038116.